log/logging_test.ts

```diff
@@ -0,0 +1,181 @@
+import {
+    buildDefaultLogMessage,
+    buildLogger,
+    defaultDispatch,
+    LogHandler,
+    toLoggableString,
+} from "./logging.ts";
+import { assert, assertEquals, assertMatch } from "../testing/asserts.ts";
+
+const testLevels = {
+  low: 1,
+  middle: 2,
+  high: 3,
+};
```

Comment:
Would be great to have some subprocess that tests handler using `console.log`

Reply:
Yes, I am working on how to do that elegantly